src/main/java/roomescape/Controller/TimeReservationController.java

```diff
@@ -0,0 +1,52 @@
+package roomescape.Controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.DTO.TimeRequestDTO;
+import roomescape.DTO.TimeResponseDTO;
+import roomescape.Domain.Time;
+import roomescape.Repository.TimeRepository;
+import roomescape.Service.TimeServiceImpl;
+
+import java.net.URI;
+import java.util.List;
+
+@Controller
+@RequestMapping("/times")
+public class TimeReservationController {
+    private final TimeServiceImpl service;
```

Comment:
```suggestion
    private final TimeService service;
```
인터페이스를 타입으로 놓는 건 어떨까요?

Reply:
찾아보니 인터페이스를 타입으로 두면 추후 수정에 더 용이하다고 하네요!
참고하여 수정해보겠습니다😄